### ISSUE TITLE
Excess attributes not detected in JSX with @types/React

### ISSUE THREAD

**Issue by RyanCavanaugh** [Microsoft/TypeScript#9958]
> npm install @types/react
> tsc --init --jsx preserve

a.tsx
```ts
import * as React from 'react';
const i = <img styl={null} />
```
Expected: Error (styl is not on img)
Actual: No error

**Comment by RyanCavanaugh** [comment]
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/types-2.0/react/index.d.ts#L2041
